### PR TITLE
Update principles-of-operation.md

### PR DESCRIPTION
**Pull request by TinaUll**
Hi @bballentine123 and @cmorgan-osi, I updated the Principles of operation topic with the data quality information (as I understood it). Please let me know if this works or if it should be reworded. Thanks and Happy Friday!

### CODE REVIEW

**Comment by TinaUll** [comment]
Hi @bballentine123 and @cmorgan-osi, do you mind reviewing the PR so I can merge? Thank you!

**Review by cmorgan-osi — APPROVED** [review]

File: content/overview/supported-features.md [hunk 1]
```diff
@@ -22,6 +22,19 @@ You can configure PI Adapter for DNP3 to read data from each of the following po
...
+**Note:** Variation zero has a special meaning in DNP3. The adapter can be configured to request a variation zero for static scans, and the outstation should return data in a variation that it prefers. The outstation is not allowed to specify variation zero in its response. The adapter will not request a specific variation for event data, but it is able to parse any of the variations listed above.
+
+## Data Quality - Object Flags
+
+Many of the supported object variations include object flags. PI Adapter for DNP3 will interpret these object flags as Data Quality and will send them as a 1 byte integer for the `Quality` property on the stream.
+
+Each bit in the object flag indicaes a separate condition. Below is a table description of the flags for the supported types. Please refer to the DNP3 Protocol Specification for more information about object flags.
```
Comment (bballentine123) [inline content/overview/supported-features.md:31]: Don't want to be nitpicky, but 'indicates' is spelled wrong

Comment (TinaUll) [reply content/overview/supported-features.md:31]: @bballentine123 I just got off the call with Cody and did not even have a chance to review. You are too fast! :)

Comment (cmorgan-osi) [inline content/overview/supported-features.md:31]:
```suggestion
Each bit in the object flag indicates a separate condition. Below is a table description of the flags for the supported types. Please refer to the DNP3 Protocol Specification for more information about object flags. 
```

To address Bennett's comment. Nice catch, @bballentine123

**Review by cmorgan-osi — APPROVED** [review]

**Review by bballentine123 — APPROVED** [review]

**Pull request merged**